msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp

```diff
@@ -95,6 +95,10 @@
 					});
 				}
 
+				$('#revise\\:forum_locked, #revise\\:moderated, #revise\\:postFirst').each(function() {
+					$(this).attr('aria-labelledby', 'forum_posting_head ' + $(this).attr('id') + '_label');
+				});
+
```

Comment:
This can be done directly in the JSF markup now that we're on a newer version of JSF. All that's needed is to add the passthrough lib import on the page:

```
<html xmlns="http://www.w3.org/1999/xhtml"
      xmlns:h="http://java.sun.com/jsf/html"
      xmlns:p="http://java.sun.com/jsf/passthrough">
```

Then you can use the prefix to add any arbitrary attribute on the tag that you don't want to be processed by the JSF rendering engine:

```
<h:outputLabel id="forum_locked_label" value="#{msgs.cdfm_lock_forum}" p:aria-labelledby="forum_posting_head...." />
```

Reply:
Hi @bjones86 , tested your approach in Forums and doesn't work :(

Reply:
Hmm... that's too bad. I wonder why it doesn't work? It would be nice not to have to use JavaScript just to attach an attribute.